core/services/workflows/engine_test.go

```diff
@@ -23,15 +23,15 @@ import (
 
 const hardcodedWorkflow = `
 triggers:
-  - id: "mercury-trigger"
+  - id: "mercury-trigger@1"
```

Comment:
You're talking about staging, right?
I think the non-breaking change would be to update the staging env such that the id entries in staging contain the version too. WDYT?

Reply:
Yes, but I'm also talking about the various CapabilityInfo objects kicking around. By modifying this without modifying those, aren't we ensuring that we won't be able to fetch existing capabilities (since new workflows will fail validation without a version?)

I would love for us to make these changes in a way that maintains functionality -- my main concern here is ensuring that what is on develop is always in a working state.